.github/workflows/windows.yml

```diff
@@ -178,10 +178,10 @@ jobs:
           flags: unit
       -
         name: Upload reports
-        if: matrix.os == 'windows-2022'
+        if: always()
```

Comment:
Should we even keep the `if` here if it's always true?

Reply:
`if: always()` is slightly different from the default - this makes this step run even if other steps fail (but the overall job status still says failure)

Reply:
We _always_ want to upload reports even if steps in the job have failed. Pretty much like in the Jenkinsfile with: https://github.com/moby/moby/blob/caf9c35e5cf05022f340ea1cf3c05ce597763852/Jenkinsfile#L1237-L1238